great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -163,11 +184,34 @@ def _count_positive_and_negative_example_cases(
         negative_cases: int = 0
 
         for test_data_cases in examples:
-            for test in test_data_cases["tests"]:
-                if test["output"]["success"] is True:
-                    positive_cases += 1
-                elif test["output"]["success"] is False:
-                    negative_cases += 1
+            try:
+                for test in test_data_cases["tests"]:
+                    if test["output"]["success"] is True:
+                        positive_cases += 1
+                    elif test["output"]["success"] is False:
+                        negative_cases += 1
+            except:
+                # If there's no "success" key, should it count for negative_cases?
+                pass
+
+                # Some examples of test["output"] with no key success
```

Comment:
I just included these in the draft PR to show examples of `test["output"]` having no "success" key